components/00-settings/_colors.scss

```diff
@@ -22,6 +22,7 @@ $colors: (
   'yellow': #ffdb5a,
   'cyan': #009de0,
   'cyan-hover': #1abcff,
+  'cyan-pastel': #7ed9ff,
```

Comment:
We kunnen de pastelkleur (dicht in de buurt) afleiden uit de base color cyan `color('cyan', -3);`. Zo moeten we geen nieuwe base colors definiëren, die op zich ook weer apart in de stijlgids zouden moeten komen + alle afgeleiden.